### PR TITLE
Add missing endpoint to the all-in-one docker container

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Port | Protocol | Component | Function
 6832 | UDP      | agent     | accept jaeger.thrift over binary thrift protocol
 5778 | HTTP     | agent     | serve configs
 16686| HTTP     | query     | serve frontend
+14268 | TCP     | collector | TCP collector endpoint
```
Comment (pavolloffay) [inline docs/getting_started.md]: Can you please change it to HTTP?

Comment (geoand) [reply docs/getting_started.md]: Sure, no problem

**Comment by pavolloffay** [comment]
It looks like `gandrian@redhat.com` is not linked with your github account.

**Comment by geoand** [comment]
@pavolloffay Should be fixed now

File: docs/getting_started.md (hunk 1 above)
Comment (pavolloffay) [reply docs/getting_started.md]: Sorry, I am looking at other tables could you please add this to the last column  `accept jaeger.thrift directly from clients` or something similar encorporating jaeger.thrift :)

Comment (geoand) [reply docs/getting_started.md]: Sure thing!

Comment (geoand) [reply docs/getting_started.md]: PR rebased and includes the `jaeger.thrift` :)

Comment (pavolloffay) [reply docs/getting_started.md]: lol it says that `zipkin.thrift`

Comment (geoand) [reply docs/getting_started.md]: Ouch... Stupid mistake :P. Sorry! Will fix ASAP

Comment (geoand) [reply docs/getting_started.md]: Should be correct this time, unless the fat fingers once again got the best of me :)

**Comment by pavolloffay** [comment]
thanks @geoand

**Comment by geoand** [comment]
You are welcome!

**Pull request merged**